### PR TITLE
feature. LB theme adjustments

### PR DESCRIPTION
**Pull request by mkdok**
- Added new Drupal modules to extend the functionality
- Adjust the appearance of the LB

### CODE REVIEW

**Comment by mkdok** [comment]
@podarok @anpolimus Could you check my changes?

File: composer.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
         "drupal/bootstrap_styles": "*",
         "drupal/bootstrap_layout_builder" : "*",
         "drupal/layout_builder_blocks" : "*",
+        "drupal/layout_builder_restrictions" : "*",
+        "drupal/lb_claro" : "*",
```
Comment (podarok) [inline composer.json:10]:
```suggestion
        "drupal/layout_builder_restrictions" : "^2.16",
        "drupal/lb_claro" : "^1.0",
```

File: y_lb.libraries.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+layout_builder:
+  version: VERSION
```
Comment (podarok) [inline y_lb.libraries.yml:2]:
```suggestion
  version: 1.0
```

**Review by podarok — CHANGES_REQUESTED** [review]
See my suggestion, approve them if no concerns, please @mkdok

**Pull request merged**